dnf5/commands/download/download.cpp

```diff
@@ -97,6 +100,12 @@ void DownloadCommand::set_argument_parser() {
     url->set_const_value("true");
     url->link_value(url_option);
 
+    auto urlallmirrors = parser.add_new_named_arg("allmirrors");
+    urlallmirrors->set_long_name("allmirrors");
+    urlallmirrors->set_description _("When running with --url, prints URLs from all available mirrors");
```

Comment:
I think you forgot parentheses at set_description call: `urlallmirrors->set_description (_("When..."));`